_preload.lua

```diff
@@ -224,6 +224,12 @@
 		tokens = true,
 	}
 	
+	api.register {
+		name = "androidapplibname",
```

Comment:
libname is a better name in this case, androidapplibname is just too hard to read